src/components/app.jsx

```diff
@@ -31,7 +32,7 @@ class App extends Component {
       <BrowserRouter>
         <main className="main">
           <Switch>
-            <Route exact path="/scoresheets" component={ScoresheetsContainer} />
+            <Route exact path="/scoresheets" component={MyScoresheetsContainer} />
```

Comment:
Should this be a ProtectedRoute since it expects a currentUser?

Reply:
Resolved!